week-1-practise/level-1/8-function-output-1.js

```diff
@@ -1,12 +1,16 @@
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
 var result = getNumber();
 console.log(result);
 
 // Answer these questions:
 // - What does this program do?
+// This program returns a random number
```

Comment:
What is the range that the returned number is in? Between -1000 and 1000? Or 0 and 1000?